Mathlib/RepresentationTheory/Rep.lean

```diff
@@ -355,33 +324,16 @@ protected def ihom (A : Rep k G) : Rep k G ⥤ Rep k G where
 def homEquiv (A B C : Rep k G) : (A ⊗ B ⟶ C) ≃ (B ⟶ (Rep.ihom A).obj C) where
   toFun f :=
     { hom := ModuleCat.ofHom <| (TensorProduct.curry f.hom.hom).flip
-      comm := fun g => by
-        ext x : 2
-        refine LinearMap.ext fun y => ?_
-        change f.hom (_ ⊗ₜ[k] _) = C.ρ g (f.hom (_ ⊗ₜ[k] _))
-        rw [← hom_comm_apply]
-        change _ = f.hom ((A.ρ g * A.ρ g⁻¹) y ⊗ₜ[k] _)
-        simp only [← map_mul, mul_inv_cancel, map_one]
-        rfl }
+      comm := fun g => ModuleCat.hom_ext <| LinearMap.ext fun x => LinearMap.ext fun y => by
```

Comment:
Does `comm g := ...` work? (And the same comment about ext.)

Reply:
Thanks for the comments! I've applied them - indeed some of the `ext`-y lines can be replaced by `ext`.